packages/media-utils/package.json

```
@@ -0,0 +1,35 @@
+{
+	"name": "@wordpress/media-utils",
+	"version": "0.1.0",
+	"description": "WordPress Media Upload Utils.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"media",
+		"upload",
+		"media-upload"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/master/packages/media-utils/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/url"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
```

React Native should also have a way to override this implementation:
```suggestion
	"module": "build-module/index.js",
	"react-native": "src/index",
```